doc/developer/workflow.rst

```diff
@@ -160,6 +160,34 @@ releases have support for this feature request. Moreover, introducing features
 requests may result in breaking the stability of the branch. LTS branches are first
 done to bring long term support for stability.
 
+Development Branches
+--------------------
+
+Occassionally the community will desire the ability to work together
+on a feature that is considered useful to FRR.  In this case the
+parties may ask the Maintainers for the creation of a development
```

Comment:
It would be cool defining in what kind of form this should be asked the Maintainers for the creation of a development branch. Mail, Slack, Github issue, or what?

Reply:
I don't think that level of spelling out is necessary.  If you are already involved in the community this will already be part of the ongoing daily talk that goes on